src/library/scala/util/Using.scala

```diff
@@ -372,14 +372,5 @@ object Using {
     *
     * @tparam R the type of the resource
     */
-  trait Releasable[-R] {
-    /** Releases the specified resource. */
-    def release(resource: R): Unit
-  }
-
-  object Releasable {
-    /** An implicit `Releasable` for [[java.lang.AutoCloseable `AutoCloseable`s]]. */
-    implicit val autoCloseableIsReleasable: Releasable[AutoCloseable] = (resource: AutoCloseable) => resource.close()
-  }
-
+  type Releasable[-R] = AutoCloseable[R]
```

Comment:
This type alias is invalid/not correct. `j.l.AutoCloseable` is not parameterised, and is in fact not a typeclass like `Releasable` is.

Reply:
You're right. If we go with `j.l.AutoCloseable` route, we should just drop the type alias and replace `:  Releasable` as `<: j.l.AutoCloseable`.